e2e/e2etests/e2etests.go

```diff
@@ -65,7 +65,7 @@ const (
 	TestSPLDepositName                         = "spl_deposit"
 	TestSPLDepositAndCallName                  = "spl_deposit_and_call"
 	TestSPLWithdrawName                        = "spl_withdraw"
-	TestSPLWithdrawAndCreateReceiverAtaName    = "spl_withdraw_and_create_receiver_ata"
+	TestSPLWithdrawFailsIfNoReceiverAtaName    = "spl_withdraw_fails_without_receiver_ata"
```

Comment:
Both name and variable names need to be consistent, let's call it `spl_withdraw_no_receiver_ata` for simplicity, the fact it should fail is part of the test assertion, in the future if it's not supposed to fail another we should be ablet to update the test without renaming it

Reply:
renamed https://github.com/zeta-chain/node/pull/3231/commits/99fdb32958065201558a9627928548ed36e09ec9